enh(mc): Great Reboot - network-hp-standard-snmp (#2874)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md

| Service Alias              | Service Template                                       | Service Description                                             | Discovery  |
|:---------------------------|:-------------------------------------------------------|:----------------------------------------------------------------|:----------:|
| Cast-Generic-Name          | Net-HP-Standard-Cast-Generic-Name-SNMP-custom          | Check packets casting type of a network interface               |            |
| Cast-Global                | Net-HP-Standard-Cast-Global-SNMP-custom                | Check packets casting type of multiple network interfaces       | X          |
| Packet-Errors-Generic-Name | Net-HP-Standard-Packet-Errors-Generic-Name-SNMP-custom | Check packets on errors/discards of a network interface         |            |
| Packet-Errors-Global       | Net-HP-Standard-Packet-Errors-Global-SNMP-custom       | Check packets on errors/discards of multiple network interfaces | X          |
| Spanning-Tree              | Net-HP-Standard-SpanningTree-SNMP-custom               | Check Spanning Tree state on interfaces                         | X          |

> | Traffic-Generic-Id         | Net-HP-Standard-Traffic-Generic-Id-SNMP-custom         | Check traffic of an network interface                           |            |

Reviewer: ```suggestion
| Traffic-Generic-Id         | Net-HP-Standard-Traffic-Generic-Id-SNMP-custom         | Check traffic of a network interface                           |            |
```

> | Traffic-Generic-Name       | Net-HP-Standard-Traffic-Generic-Name-SNMP-custom       | Check traffic of an network interface                           |            |

Reviewer: ```suggestion
| Traffic-Generic-Name       | Net-HP-Standard-Traffic-Generic-Name-SNMP-custom       | Check traffic of a network interface                           |            |
```

#### Host discovery

| Rule name       | Description                                                                                                                                                      |
|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule |

More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).

#### Service discovery

| Rule name                               | Description                                                           |
|:----------------------------------------|:----------------------------------------------------------------------|
| Net-Hp-Standard-SNMP-Cast-Name          |                                                                       |
| Net-Hp-Standard-SNMP-Packet-Errors-Name | Discover network interfaces and monitor errored and discarded packets |

> | Net-Hp-Standard-SNMP-Spanning-Tree-Name |                                                                       |

Reviewer: description?

You still have to manually install the plugin on the poller(s) when:
- Automatic plugin installation is turned off
- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet

> More information in the [Installing the plugin](/docs/monitoring/pluginpacks/#installing-the-plugin) section.

Use the commands below according to your operating system's package manager:

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```bash
dnf install centreon-plugin-Network-Hp-Standard-Snmp
```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```bash
dnf install centreon-plugin-Network-Hp-Standard-Snmp
```

</TabItem>
<TabItem value="Debian 11" label="Debian 11">

```bash
apt install centreon-plugin-network-hp-standard-snmp
```

</TabItem>
<TabItem value="CentOS 7" label="CentOS 7">

```bash
yum install centreon-plugin-Network-Hp-Standard-Snmp
```

</TabItem>
</Tabs>

## Using the monitoring connector

### Using a host template provided by the connector

1. Log into Centreon and add a new host through **Configuration > Hosts**.

> 2. Fill the **Name**, **Alias** & **IP Address/DNS** fields according to your ressource settings.

Reviewer: ```suggestion
2. Fill in the **Name**, **Alias** & **IP Address/DNS** fields according to your resource's settings.
```
